src/types/RequestPayload.ts

```diff
@@ -5,16 +5,16 @@ import { JSONObject } from './JSONObject';
  *
  * @see https://docs.kuzzle.io/core/2/api/payloads/request
  */
-export type RequestPayload = {
+export interface RequestPayload {
```

Comment:
Don't forget to update this page: https://docs.kuzzle.io/core/2/framework/types/request-payload/

Reply:
Unfortunately, this is in Kuzzle doc :roll_eyes: I will open a separate PR